app/services/hyrax/admin_set_create_service.rb

```diff
@@ -74,7 +74,21 @@ def access_grants_attributes
       ].tap do |attribute_list|
         # Grant manage access to the creating_user if it exists. Should exist for all but default Admin Set
         attribute_list << { agent_type: 'user', agent_id: creating_user.user_key, access: Hyrax::PermissionTemplateAccess::MANAGE } if creating_user
+      end + managers_of_admin_set
```

Comment:
This is the behavior change to copy the managers from the admin set collection type to the admin set when it is created.